lab_5_scrapper/scrapper.py

```diff
@@ -13,70 +73,116 @@ def __init__(self, path_to_config: Path) -> None:
         """
         Initializes an instance of the Config class
         """
-        pass
+        self.path_to_config = path_to_config
+        self.config_content = self._extract_config_content()
+        self._seed_urls = self.config_content.seed_urls
+        self._num_articles = self.config_content.total_articles
+        self._headers = self.config_content.headers
+        self._encoding = self.config_content.encoding
+        self._timeout = self.config_content.timeout
+        self._should_verify_certificate = self.config_content.should_verify_certificate
+        self._headless_mode = self.config_content.headless_mode
 
     def _extract_config_content(self) -> ConfigDTO:
         """
         Returns config values
         """
-        pass
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config_data = json.load(file)
+        return ConfigDTO(**config_data)
 
     def _validate_config_content(self) -> None:
         """
         Ensure configuration parameters
         are not corrupt
         """
-        pass
+        with open(self.path_to_config, 'r', encoding='utf-8') as file:
+            config_data = json.load(file)
+
+        seed_urls = config_data.get('seed_urls')
+        headers = config_data.get('headers')
+        total_articles_to_find_and_parse = config_data.get('total_articles_to_find_and_parse')
+        encoding = config_data.get('encoding')
+        timeout = config_data.get('timeout')
+        should_verify_certificate = config_data.get('should_should_verify_certificate')
+        headless_mode = config_data.get('headless_mode')
+
+        if not isinstance(seed_urls, list):
+            raise IncorrectSeedURLError
+
+        for seed_url in seed_urls:
+            if not re.match(r'https?://w?w?w?.', seed_url):
+                raise IncorrectSeedURLError
+
+        if not isinstance(total_articles_to_find_and_parse, int) or isinstance(
+                total_articles_to_find_and_parse, bool) \
+                or total_articles_to_find_and_parse < 0:
+            raise IncorrectNumberOfArticlesError
+
+        if total_articles_to_find_and_parse < 1 or total_articles_to_find_and_parse > 150:
```

Comment:
one line above you check to be non negative, now only positive. can you join conditions because they duplicate each other

Reply:
done